spec/classes/shared_base.rb

```diff
@@ -11,14 +11,26 @@
   end
 end
 
+shared_examples_for :base_8 do
+  include_context :base
+
+  it do
+    is_expected.to contain_yumrepo('epel').with(
+        mirrorlist: 'https://mirrors.fedoraproject.org/metalink?repo=epel-8&arch=$basearch',
+        gpgkey:     'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-8',
+        descr:      'Extra Packages for Enterprise Linux 8 - $basearch'
+    )
+  end
+end
+
 shared_examples_for :base_7 do
   include_context :base
 
   it do
     is_expected.to contain_yumrepo('epel').with(
-      mirrorlist: 'https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch',
-      gpgkey:     'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-EPEL-7',
-      descr:      'Extra Packages for Enterprise Linux 7 - $basearch'
+        mirrorlist: 'https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch',
```

Comment:
…and an extra two spaces of indent here, as well as many other places.

Reply:
All cases fixed.